zebra/rt_netlink.c

```diff
@@ -2359,6 +2360,12 @@ static void netlink_handle_5549(struct ndmsg *ndm, struct zebra_if *zif,
 	if (ipv4_ll.s_addr != ip->ip._v4_addr.s_addr)
 		return;
 
+	if (handle_failed && ndm->ndm_state & NUD_FAILED) {
+		zlog_info("Neighbor Entry for %s has entered a failed state, not reinstalling",
```

Comment:
should probably print the IP address here too?  pretty confusing to get a message about a whole interface failing...

Reply:
if we have gotten to this point the whole interface has failedc.

Reply:
to be pedantic -> it's running bgp unnumbered (and by it's nature it's ptp ) and our installed neighbor entry is not being rejected when we install it, the kernel is sending us a Failure message.  So the interface is sufficient in this case.  ( if we see this we need to figure out what has gone wrong on the interface )